pytest_pyodide/runner.py

```diff
@@ -449,7 +449,13 @@ def get_driver(self, jspi=False):
         options = Options()
         options.add_argument("--headless")
         options.add_argument("--no-sandbox")
-        if jspi:
+        driver = Chrome(options=options)
+        chrome_version = int(
+            driver.capabilities.get("browserVersion", "0").split(".")[0]
+        )
+        driver.quit()
```

Comment:
I am curious how expensive it is to initialize a Chrome instance here. If it actually runs a new Chrome browser, it would be nice if we can cache the value somewhere instead of calling it everytime in get_driver